src/components/hooks/use-firebase-fetch-data.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react'
+import { initializeApp } from 'firebase/app'
+import { getDatabase, ref, onValue } from 'firebase/database'
+import { firebaseConfig } from 'common/constants'
+import useRegion from 'components/hooks/use-region'
+
+const useFetchFirebaseData = () => {
```

Comment:
Can we rename this to?

```suggestion
const usePricingFeed = () => {
```